### DISCUSSION TITLE
Delete a project

### DISCUSSION THREAD

**Discussion by teger** [WeblateOrg/weblate#9004] [General]
Hi,

i would like to delete a project in django interface !
But i have a group automatically created by the weblate and i don't have sufficient right to delete this group !

But i logged with the admin user !!

How to you do to delete a group please ?

Thanks for your help

**Comment by nijel** [comment]
Do you want to delete a group or a project? Built-in teams cannot be removed, see https://docs.weblate.org/en/latest/admin/access.html#list-of-teams

PS: Using Django admin interface is going to be deprecated, see https://github.com/WeblateOrg/weblate/issues/2363

**Comment by teger** [comment]
Hi,

I want to delete a project.
But in weblate, it writes :  has been scheduled for deletion.
But the project is never deleted !

And in django interface, it asks to delete the project to delete before a group automatically created with the project !


Thanks for your help

Best regards

**Reply by nijel** [reply]
Probably Celery is not running and thus background tasks are never executed. See https://docs.weblate.org/en/latest/admin/install.html#background-tasks-using-celery
